Add metadata for MIT Compass podcast

pdpinch opened the pull request:
### What are the relevant tickets?
<!--- If it fixes an open issue, please link to the issue here. -->
<!--- Closes # --->
<!--- Fixes # --->

response to Appzi suggestion: https://portal.appzi.com/portals/Q2pSI/feedback/ec8c7495-2355-4968-9141-95be2aa8507b

### Description (What does it do?)
<!--- Describe your changes in detail -->

Please add the MIT Compass podcast to Learn ❨as its MITx version is starting tomorrow❩

pdpinch committed:
Add metadata for MIT Compass podcast

pdpinch commented:
@abeglova can you take a look at this?

abeglova commented on podcasts/mit-compass:
```diff
@@ -0,0 +1,6 @@
+---
+rss_url: https://compass.mit.edu/feed/
+topics: Philosophy
+offered_by: "MIT School of Humanities, Arts, and Social Sciences"
+website: https://compass.mit.edu/podcast/
+apple_podcasts_url: https://podcasts.apple.com/us/podcast/mit-compass/id1822301964
```
this should be a .yaml file

abeglova commented:
the import worked great as is locally but the file extension should be changed to .yaml to match the other files in the folder

pdpinch committed:
use correct .yaml file extension

pdpinch committed:
Add Google Podcasts URL to MIT Compass podcast

The Google Podcasts site has moved to YouTube. I'm not sure what the canonical URL for a podcast should be, but this one seems to work well.

pdpinch committed:
Update Google Podcasts URL in mit-compass.yaml

This link goes directly to the podcast playlist.
